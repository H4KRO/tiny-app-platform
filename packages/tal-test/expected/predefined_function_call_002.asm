main()
  entry:
    Literal 0
    MakeArray
    Literal 0
    MakeRecord
    Local             name: give_hello
    Call
    Literal " Toto"
    Intrinsic         operation: INTRINSIC_ADD
