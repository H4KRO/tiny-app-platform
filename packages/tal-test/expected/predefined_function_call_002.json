[
  {
    "kind": "BinaryOperator",
    "operator": "+",
    "right": " Toto",
    "left": {
      "kind": "Call",
      "args": [],
      "value": {
        "kind": "Local",
        "name": "give_hello"
      }
    }
  }
]